src/main/resources/jar_mode_min_program_compilation_status.json

```diff
@@ -0,0 +1,22 @@
+{
+  "cf-1291": "FAIL",
```

Comment:
Are these accurate? If so, that's very concerning.

Reply:
Projects where I could not write the script to pull dependencies, specimin was not run on them and results are showing as  FAIL.
specimin was not executed on cf-1291 in jar mode. Thats why status is FAIL